iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -116,10 +143,10 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
                 _closed = true;
             }
 
+            cancellationToken.ThrowIfCancellationRequested();
             try
             {
-                cancellationToken.ThrowIfCancellationRequested();
-
+                _twinTimeoutTimer.Dispose();
```

Comment:
If we create it on ctor and dispose on closeasync, we won't be able to call openasync again. We need to build/destroy it at the same type of event.

Reply:
so maybe dispose in the dispose method, and simply stop the timer and start it on open/close?

Reply:
Disposing it in Dispose method.
There is no way to close the time though